servo-tidy.toml

```diff
@@ -68,6 +68,9 @@ packages = [
     "phf_codegen",
     "phf_generator",
     "phf_shared",
+
+    # Duplicated by gstreamer/servo-media
+    "toml_edit",
```

Comment:
It looks like you could avoid this by updating `toml` in `support/filterlock/Cargo.toml` to "0.8.9".